scripts/ai/jobs/CpAIJobCombineUnloader.lua

```diff
@@ -63,16 +62,14 @@ function CpAIJobCombineUnloader:applyCurrentState(vehicle, mission, farmId, isDi
 		end
 	end
 
+	x, z = vehicle:getCpCombineUnloaderFieldPosition()
```

Comment:
@schwiti6190 why is this different/better than the code segment above with lastJob?

Reply:
The idea is to save the position of the field marker into the vehicle(job), so it can be saved and reloaded.
The in game menu works with blank new Jobs, everytime it's opened. 
This deletes the selected field position on reopening of the job.
The fieldwork/balefinder jobs allways use the current vehicle position as reference for the field selection.